server/app.py

```diff
@@ -49,6 +50,8 @@ def get_app(config=None):
 
         ElasticAPM(app)
 
+    app.jinja_env.globals.update(group_items_by_state=group_items_by_state)
```

Comment:
This will do fine for now. As an improvement for later (maybe for Superdesk v2.2) it might be better to have a `register_jinja_function` in the core
@petrjasek